eng/scripts/InstallProcDump.ps1

```diff
@@ -0,0 +1,46 @@
+<#
+.SYNOPSIS
+    Installs ProcDump into a folder in this repo.
+.DESCRIPTION
+    This script downloads an extracts the ProcDump.
+.PARAMETER Force
+    Overwrite the existing installation
+#>
+param(
+    [switch]$Force
+)
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue' # Workaround PowerShell/PowerShell#2138
+
+Set-StrictMode -Version 1
+
+$repoRoot = Resolve-Path "$PSScriptRoot\..\.."
+$installDir = "$repoRoot\.tools\ProcDump\"
+$tempDir = "$repoRoot\obj"
+
+if (Test-Path $installDir) {
+    if ($Force) {
+        Remove-Item -Force -Recurse $installDir
+    }
+    else {
+        Write-Host "ProcDump already installed to $installDir. Exiting without action. Call this script again with -Force to overwrite."
+        exit 0
+    }
+}
+
+Remove-Item -Force -Recurse $tempDir -ErrorAction Ignore | out-null
+mkdir $tempDir -ea Ignore | out-null
+mkdir $installDir -ea Ignore | out-null
+Write-Host "Starting ProcDump download"
+Invoke-WebRequest -UseBasicParsing -Uri "https://download.sysinternals.com/files/Procdump.zip" -Out "$tempDir/ProcDump.zip"
```

Comment:
We do this in several places in the pipeline so I don't want to tackle that as part of this PR as it will unnecessarily slow it down and it can be done in parallel. Instead, I filed an issue to fix this for all the places where we download an external asset. See https://github.com/aspnet/AspNetCore-Internal/issues/3125 for details

Reply:
Glad you filed the issue. Have you started the conversation w/ the Arcade folks about making procDump available?

Reply:
I expect the issue to follow the normal triage process and that action is taken when it gets scheduled based on priority.